typescript/packages/wallets/sui/src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { type SuiClient } from "@mysten/sui.js/client";
+
+export type AwesomeChainResponse = unknown;
```

Comment:
No, this should be something like:

`SuiReadResponse`

and include a field called `object` that includes the object returned